includes/Settings.php

```diff
@@ -287,4 +288,15 @@ private function doIterate( $key, $options ) {
 		throw new SettingNotFoundException( "'{$key}' is not a valid settings key" );
 	}
 
+	private static function initLegacyMapping( &$configuration ) {
+
+		// When adding a deprecated mapping, extend the DeprecationNoticeTaskHandler
+		// with the corresponding entry
+
+		// Deprecated in 2.5, to be removed with 3.1
```

Comment:
> Removal in a feature release?

Well, there are two releases a part. Furthermore, this consolidates the literal description (found in release notes, setting notes etc.) to an actional representation by making #2357 to detect those and inform the user more actively about pending future removal.

See also https://github.com/SemanticMediaWiki/SemanticMediaWiki/issues/2343#issuecomment-289252430.